HIP/hip-1084.md

```diff
@@ -0,0 +1,118 @@
+---
+hip: 1084
+title: Zero Cost EthereumTransaction on Success
+author: Nana Essilfie-Conduah <@Nana-EC>, Richard Bair <@rbair>
+working-group: Richard Bair <@rbair>, Atul Mahamuni <@atul-hedera>, Stanimir Stoyanov <stanimir.stoyanov@limechain.tech>
+requested-by: Relay Operators
+type: Standards Track
+category: Service
+needs-council-approval: Yes
+status: Draft
+created: 2024-11-20
+discussions-to: https://github.com/hashgraph/hedera-improvement-proposal/discussions/1083
+updated: 2024-12-06
+requires: 410
+---
+
+## Abstract
+
+Updates the rules for charging fees for EthereumTransactions, so that the payer of an `EthereumTransaction` will only
+incur a fee if the transaction fails due diligence checks.
+
+## Motivation
+
+Today, relay operators must pay the `EthereumTransaction` HAPI fee for every transaction they submit on behalf of an
+Externally Owned Account (EOA). Currently, for every simple `EthereumTransaction`, this fee is the hbar equivalent of
+$0.0001, based on the exchange rate. Although this fee is very low, it effectively compels relay operators on Hedera
+to adopt a non-standard business model to cover these costs, which are then passed on to the users. Users are also
+burdened with understanding this more complex business model, which negatively impacts their experience.
+
+This situation acts as a friction point for EOAs looking to access cost-effective relay operators on the Hedera public
+network.
+
+## Rationale
+
+On Ethereum, all execution costs are paid for by the 'sender' who created, signed, and submitted the transaction. Hedera
+follows a similar approach for all `ContractCallTransaction`s but also charges an additional fee for each
+`EthereumTransaction` submitted via JSON-RPC relays. This fee serves as a defense against a malicious JSON-RPC relay that
+might send malformed transactions to the network, where there would be no one to pay for the processing of those transactions.
+This fee is unnecessary for successful transactions since the intrinsic gas fee covers the base costs (gossip & consensus).
+Therefore, this HIP proposes that the fee for submitting `EthereumTransactions` to the network should only be charged if the
+JSON-RPC relay submits a transaction that fails due diligence checks, such that the EOA cannot be charged for the overhead
+of gossip and consensus.
```

Comment:
```suggestion
On Ethereum, all execution costs are paid for by the 'sender' who created, signed, and submitted the transaction.
Hedera follows a similar approach for `ContractCallTransaction`s. However, Hedera charges an additional fee for
each `EthereumTransaction` submitted via JSON-RPC relays. This fee serves as a defense against a malicious
JSON-RPC relay that might send malformed transactions to the network, where there would be no one to pay for
the processing of those transactions.

This fee is unnecessary for successful transactions since the intrinsic gas fee covers the base costs (gossip &
consensus). Therefore, this HIP proposes that the fee for submitting `EthereumTransactions` to the network
should only be charged if the JSON-RPC relay submits a transaction that fails due diligence checks, such that the
EOA cannot be charged for the overhead of gossip and consensus.
```

Reply:
Changes applied